x-pack/plugins/cloud/README.md

```diff
@@ -1,3 +1,12 @@
 # `cloud` plugin
 
 The `cloud` plugin adds cloud specific features to Kibana.
+The plugin configures following values: 
```

Comment:
NIT: The list below looks like it's the Public `setup` contract. The Server-side contract is slightly different.

Reply:
You're right, @afharo! I haven't thought about the server-side and never used it before, so I added 'client-side' to the text.